geopandas/base.py

```diff
@@ -830,6 +830,29 @@ def make_valid(self):
         """
         return _delegate_geo_method("make_valid", self)
 
+    def segmentize(self, max_segment_length):
+        """Adds vertices to line segments based on maximum segment
+        length. Returns a ``Geoseries`` .
+        Additional vertices will be added to every line segment
+        in an input geometry so that segments are no longer than
+        the provided maximum segment length.
+        New vertices will evenly subdivide each segment.
+        Only linear components of input geometries are densified;
+        other geometries are returned unmodified.
```

Comment:
```suggestion
        """Adds vertices to line segments based on maximum segment length.

        Additional vertices will be added to every line segment in an input geometry so
        that segments are no longer than the provided maximum segment length. New
        vertices will evenly subdivide each segment. Only linear components of input
        geometries are densified; other geometries are returned unmodified. Returns a
        ``Geoseries`` .
```

Reformatted to use all 88 characters per line.